eZ/Publish/API/Repository/ContentService.php

```diff
@@ -202,7 +203,7 @@ public function loadContentListByContentInfo(array $contentInfoList, array $lang
      *
      * @return \eZ\Publish\API\Repository\Values\Content\Content - the newly created content draft
      */
-    public function createContent(ContentCreateStruct $contentCreateStruct, array $locationCreateStructs = []): Content;
+    public function createContent(ContentCreateStruct $contentCreateStruct, array $locationCreateStructs = [], ?array $fieldIdentifiersToValidate = null): Content;
```

Comment:
While in the previous implementation new boolean argument was rather straightforward, now I'm missing a bit of PHPDoc regarding `?array $fieldIdentifiersToValidate = null`. The questions such doc needs to answer are:
* what happens when value is default `null`?
* how is that different when value is `[]`? 
* how does it work in general?

Reply:
Docblock updated.